style.css

```diff
@@ -0,0 +1,135 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-content: space-around;
+  max-width: 1024px;
+  margin: 20px auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 50px;
+}
+h1 {
```

Comment:
Please, add one empty line between different selectors. It would be more readable.

Reply:
Done!